scripts/serverless_init_dotnet.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Check if Alpine
+if [ -f "/etc/alpine-release" ]; then
+    echo "Alpine images are not supported for .NET automatic instrumentation with serverless-init"
+    exit 1
+fi
+
+# Make sure curl is installed
+apt-get update && apt-get install -y curl
+
+# Get latest released version of dd-trace-dotnet
+TRACER_VERSION=$(curl -s https://api.github.com/repos/DataDog/dd-trace-dotnet/releases/latest | jq -r '.tag_name' | cut -d '"' -f 4 | cut -c2-)
```

Comment:
What if they hit rate limit? Should we hardcode a fallback version?

Reply:
yeah good point 👍

Reply:
had this page open and just saw this and thinking it's probably safer to let the operation fail (fast) than fallback to any version which might cause unexpected issue.

Reply:
> it's probably safer to let the operation fail (fast)
wouldn't this crash your customer if it fails?

Reply:
that's a very good question. I was thinking that we should favor failing at deployment time over any fallback solution. [more about the evil of fallback](https://aws.amazon.com/builders-library/avoiding-fallback-in-distributed-systems/) 
However, this is a good question on when it's "crashing". I assumed that the image build is built once and reused when scaling happens, which is the most sensitive time.
If it's not reused, crashing during scaling may be indeed undesirable.
But could it even create a scenario where some instances running with one version and others using fallback versions?

Reply:
> But could it even create a scenario where some instances running with one version and others using fallback versions?

I don't know, you tell me!

I agree, I don't like fallbacks, yet I don't want us to crash customers. Pick as your product needs 😛

Reply:
ah yeah, that's true. It might not be so obvious to the user that there's a fallback version too should the version request fail when they're running this in their pipelines. I'll revert